### ISSUE TITLE
Fix no longer valid automated test expectation for AUEN

### ISSUE THREAD

**Issue by joshfeck** [eventespresso/eea-automated-upcoming-event-notifications#6]
Originally reported in codebase 11216:

Tests [started failing for the add-on][1] for 4.9.53.p and the current master at the time of this ticket.  The reason for the fail is that prior to the work done in #11195 getting merged in and released, this code:

```
//test content outputs for attendee
        //datetime in content
        $this->assertNotFalse(
            strpos(
                $messages['attendee']->content(),
                'We\'re reaching out to remind you of an upcoming event you registered for on our website. '
                . 'You have access to this event on '
                . $datetime->get_i18n_datetime('DTT_EVT_start')
                . ' - '
                . $datetime->get_i18n_datetime('DTT_EVT_end')
                . '. Here\'s a copy of your registration details:'
            )
        );
```
Was testing the right expectation for the value on `$messages['attendee']->content().

Currently the value on that field is _now_:

```
We&#039;re reaching out to remind you of an upcoming event you registered for on our website. You have access to this event on
```

Notice the encoded quotes.

For now, I'm going to leave the failing test because there's still ongoing work in #11211 that might change the expectation yet again.  However I just wanted to create this ticket to track the fail and make sure it eventually gets fixed (if necessary) once a resolution is figured out for #11211 (and any other related encoding issue tickets).  

I've tested, and this is not a user facing issue because _currently_ `html_entity_decode($content, ENT_QUOTES)` is run against the content field before its sent/dispatched, so the entity encoding will be taken care of.  But that's also why I want to wait for #11211 to get handled before tackling this ticket (and why its a dependency) because I don't know if expectations will change again.

After @mnelson4 and I talked about this in #11211, he pointed out that usage of `esc_html` in the default templates is what is encoding the quotes.   So it's his assessment (and I concur) that we should just change the test expectations in this ticket.  That's an easy fix that should take a couple minutes.

  [1]: https://travis-ci.org/eventespresso/eea-automated-upcoming-event-notifications/builds/309335361

**Comment by joshfeck** [comment]
Did this get fixed in #1 @darren?

**Comment by nerrad** [comment]
The [nightly tests for AUEN](https://travis-ci.org/eventespresso/eea-automated-upcoming-event-notifications) are passing so ya this ticket is not needed.
